ftw/htmlblock/profiles/default/types/ftw.htmlblock.HtmlBlock.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<object name="ftw.htmlblock.HtmlBLock"
+        meta_type="Dexterity FTI"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="ftw.htmlblock">
+
+    <!-- Basic metadata -->
+    <property name="title" i18n:translate="">HtmlBlock</property>
```

Comment:
Yeah, I already planned to use `HTML block` everywhere (`HTML-Block` in German translation).